apps/opencs/model/prefs/state.cpp

```diff
@@ -166,6 +166,16 @@ void CSMPrefs::State::declare()
         setTooltip ("When using next/previous functions at the last/first item of a "
         "list go to the first/last item");
 
+    declareCategory ("Marker Size");
+    declareDouble ("MarkerShaftWidth", "Marker Shaft Width", 30.0).
+        setRange (0.001, 100.0);
+    declareDouble ("MarkerShaftBaseLength", "Marker Shaft Base Length", 70.0).
+        setRange (0.001, 100.0);
+    declareDouble ("MarkerHeadWidth", "Marker Header Width", 50.0).
+        setRange (0.001, 100.0);
+    declareDouble ("MarkerHeadLength", "Marker Header Length", 50.0).
```

Comment:
Also, the defaults can probably be set smaller. The original values before I changed them were: 10, 50, 30, 30 in that order. It seems to be a good middle ground, so I would go with those.